x-pack/plugins/ecs_data_quality_dashboard/server/lib/data_stream/results_field_map.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FieldMap } from '@kbn/data-stream-adapter';
+
+export const resultsFieldMap: FieldMap = {
+  'meta.batchId': { type: 'keyword', required: true },
+  'meta.ecsVersion': { type: 'keyword', required: true },
+  'meta.errorCount': { type: 'long', required: true },
+  'meta.ilmPhase': { type: 'keyword', required: true },
+  'meta.indexId': { type: 'keyword', required: true },
+  'meta.indexName': { type: 'keyword', required: true },
+  'meta.isCheckAll': { type: 'boolean', required: true },
+  'meta.numberOfDocuments': { type: 'long', required: true },
+  'meta.numberOfFields': { type: 'long', required: true },
+  'meta.numberOfIncompatibleFields': { type: 'long', required: true },
+  'meta.numberOfEcsFields': { type: 'long', required: true },
+  'meta.numberOfCustomFields': { type: 'long', required: true },
+  'meta.numberOfIndices': { type: 'long', required: true },
+  'meta.numberOfIndicesChecked': { type: 'long', required: true },
+  'meta.numberOfSameFamily': { type: 'long', required: true },
+  'meta.sameFamilyFields': { type: 'keyword', required: true, array: true },
+  'meta.sizeInBytes': { type: 'long', required: true },
+  'meta.timeConsumedMs': { type: 'long', required: true },
+  'meta.unallowedMappingFields': { type: 'keyword', required: true, array: true },
+  'meta.unallowedValueFields': { type: 'keyword', required: true, array: true },
```

Comment:
Consider removing the `meta` prefix, or an alternative prefix, if the `meta` prefix, i.e. `meta.isCheckAll` in the KQL query, and the sum of `meta.numberOfIndicesChecked` in the example Lens visualization below doesn't, convey what `meta` represents:

![meta](https://github.com/elastic/kibana/assets/4459398/b2ef6129-b8cd-4bf0-8c0f-6b3590332b00)

Put differently, does the `meta` prefix describe an entity, like `cloud`, `host`, or `user` in the [ECS Field Reference](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html)?